packages/ckan/src/xml.js

```diff
@@ -130,38 +135,21 @@ const toXML = (dataset) => {
 
 const serializeTerm = (pointer) => {
   return pointer.map((value) => {
-    if (isLiteral(value)) {
-      return serializeLiteral(value)
-    } else if (isNamedNode(value)) {
-      return serializeNamedNode(value)
-    } else if (isBlankNode(value)) {
-      return serializeBlankNode(value)
-    } else {
-      return {}
-    }
+    return serializeLiteral(value) || serializeNamedNode(value) || serializeBlankNode(value) || {}
```

Comment:
Changed this to transparently replace `serializeTerm` with `serializeBlankNode` above.